src/components/Cards/components/FrontCard.tsx

```diff
@@ -1,24 +1,36 @@
-const FrontCard = () => (
+type FrontCardProps = {
+    cardholderName: string
+    cardNumber: string
+    expMonth: string
+    expYear: string
+}
+
+const FrontCard = ({ 
+    cardholderName, 
+    cardNumber,
+    expMonth,
+    expYear
+}: FrontCardProps) => (
     <div className="bg-[url('assets/images/bg-card-front.png')] bg-cover w-72 h-40 md:w-96 md:h-52 rounded-lg p-6 lg:col-start-1 lg:col-end-4 lg:static absolute top-28 left-5 sm:top-32 sm:left-6 z-10">
         <div className="text-white flex flex-col h-full justify-between">
 
             <div className="flex flex-row items-center">
-                <div className="w-7 h-7 md:w-10 md:h-10 rounded-full border-2 border-white bg-white"/>
-                <div className="w-3 h-3 md:w-5 md:h-5 rounded-full border border-white ml-2 md:ml-4"/>
+                <div className="w-7 h-7 md:w-10 md:h-10 rounded-full border-2 border-white bg-white" />
+                <div className="w-3 h-3 md:w-5 md:h-5 rounded-full border border-white ml-2 md:ml-4" />
             </div>
 
             <div className="flex flex-col h-1/2 justify-around">
                 <p className="w-full text-lg md:text-2xl text-left tracking-widest">
-                    0000 0000 0000 0000
+                    {cardNumber || '0000 0000 0000 0000'}
                 </p>
                 <div className="flex justify-between text-xs md:text-sm uppercase tracking-widest">
-                    <p>Jane Appleseed</p>
-                    <p>00/00</p>
+                    <p>{cardholderName || 'Jane Appleseed'}</p>
```

Comment:
if the name is long you will need more space between name and date.

![image](https://github.com/user-attachments/assets/edf92edd-9f01-4a6d-9b87-6aea9de217b5)

Reply:
![image](https://github.com/user-attachments/assets/f7afbf18-2e30-49ce-b395-f2e731978fdc)